### ISSUE TITLE
I2I: Publishing LTS Release Schedule

### ISSUE THREAD

**Issue by rcebulko** [ampproject/amphtml#26206]
Per https://github.com/ampproject/amphtml/issues/25578 , we're introducing a new "LTS" release channel which is updated once approximately every 4 weeks, adjusting for code freezes where needed. We should have some way to publicly provide the weeks in which releases will be promoted to LTS.

Ideas:
- Calendar/table of weeks, manually updated (infrequently) to account for code freezes
- RSS feed

Ideas that would remove the need for a public schedule:
- Promote LTS the Nth of every month
- Promote LTS every M releases
- Promote LTS the 1st ___day of each month

/cc @ampproject/wg-infra

**Comment by danielrozenberg** [comment]
I think a simple listing of dates that we update once a year would be fine, with an asterisk saying that those dates might change at the decision of the TSC for emergencies, etc (basically same asterisk that we have for the stable channel). I think it should be easy enough to figure out the listing for the whole year, and shift +1 weeks when we have release freezes (e.g. because of holidays or conferences)

**Comment by nainar** [comment]
Can't make the design review so voicing off here: 

So the following would have my vote:
- Promote LTS the Nth of every month
- Promote LTS the 1st ___day of each month

Between these depends on which aligns closest to our current workflow of releasing. I suspect it's "Promote LTS the 1st ___day of each month". 

Based on this we can use Daniel's idea of having a static list that we push to once a year.

**Comment by estherkim** [comment]
I'm for "Promote LTS the 1st ___day of each month" for simplicity and predictability, but one caveat is that it confines us to a monthly schedule. Do you foresee publishers wanting to extend the cadence to more than ~4 weeks?

Of course we can start with it, and iterate as we go.

**Comment by rcebulko** [comment]
Promote __day of the each month has predictability for publishers, which is an advantage.

There are a few drawbacks I see, the first being related to the actual day that the LTS release is promoted. For a few reasons, it makes sense to do this promote Monday:
- the release on-duty has less on their plate on Monday vs. the day of the **stable** promote
- since **lts** is promoted from the current **stable**, doing them on the same day introduces potential errors (ex. on-duty forgets that it's an LTS week, promotes the new **stable**, then promotes the new **stable** to **lts** instead of the old one

If the **lts** promote _is_ done Monday, that introduces new confusion around which weeks include an **lts** promote in cases where, for example, Monday falls on the end of one month while the Wednesday of that week is the 1st Wednesday of the month.

Given this confusion, we might need a static/manually maintained list of promotion dates even if we did opt for the 1st ___day of the month approach, which eliminates some of the simplicity savings from that idea.

I think my takeaway here is that such an approach would either introduce more ambiguity (for both publishers and AMP release on-duties) or would require the same degree of manual date publishing. I think being explicit about the dates is a positive and will be easiest to communicate both externally and within the team.

**Comment by rcebulko** [comment]
Update: During design review, Joey suggested 2nd __day of the month would avoid the ambiguity of a week split over two months (pretty much all present agreed). It would also avoid common days off (4th of July, Thanksgiving, etc.). jridgewell pointed out that Tuesday is when **beta** is promoted to **stable**, which is another point in favor of doing promotes Monday so there's no risk of accidentally promoting a new/untested **stable** to **lts**

TL;DR - LTS promotions will occur the 2nd Monday of every month. On the rare occasions where this coincides with a holiday, the promote will occur the next day (ie. Tuesday) before the current **beta** is promoted to the new **stable**.

**Comment by rcebulko** [comment]
Addressed with documentation changes and releasing 2nd Monday each month. Closing.
